vocabularies/voc4cat/0000017.ttl

```diff
@@ -6,10 +6,10 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0000017"^^xsd:token ;
     skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000016> ;
-    skos:definition "A photocatalyst introduced to the reaction chamber in the form of a powder. Usually the powdered photocatalyst is spread inside a quartz / glass plate."@en ;
+    skos:definition "A catalyst introduced to the reaction chamber in the form of a powder. Usually the powdered catalyst is spread inside a quartz / glass plate."@en ;
```

Comment:
Two comments:
- when is the catalyst a powder (vs. a small a prticle?). There has to be some size limit.
- should it be "over" instead of "inside a quartz/glass plate"

If the catalyst is attached to a plate, why is it still considered a "powdered catalyst"? Instead of for example a "catalyst-coated substrate" or "catalyic coating"?